src/factory/tests/mocks.ts

```diff
@@ -194,3 +215,43 @@ export const amMeasureHeaderItems = {
         }
     }
 };
+
+export const executionResultWithAmMeasures: Execution.IExecutionResponse = {
```

Comment:
"result" in name but "response" type. Seems inconsistent also with `emptyExecutionResponse`